core/storage/repository/impl/in_memory_repository.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef FILECOIN_CORE_STORAGE_IMPL_IN_MEMORY_REPOSITORY_HPP
+#define FILECOIN_CORE_STORAGE_IMPL_IN_MEMORY_REPOSITORY_HPP
+
+#include "storage/repository/repository.hpp"
+
+namespace fc::storage::repository {
+
+  using Version = Repository::Version;
+
+  static constexpr Version kInMemoryRepositoryVersion = 1;
+
+  /**
+   * @brief In-memory implementation of Repository
+   */
+  class InMemoryRepository : public Repository {
+   public:
+    InMemoryRepository();
+
+    ~InMemoryRepository() override = default;
```

Comment:
I think, that this default override destructor doesn't affect behaviour of this code and can be omitted
[https://stackoverflow.com/questions/40999791/default-override-of-virtual-destructor/41001264](https://stackoverflow.com/questions/40999791/default-override-of-virtual-destructor/41001264)